### PR TITLE
build: Fix Android apk build

### PR DESCRIPTION
**Pull request by aitor-lunarg**
Adds a step to add libraries in as Android likes them

### CODE REVIEW

File: build-android/build_all.sh [hunk 1]
```diff
@@ -73,7 +73,8 @@ fi
 echo "Building Layer using ${ANDROID_STL_TYPE}";
 
 function create_APK() {
-    aapt package -f -M AndroidManifest.xml -I "$ANDROID_SDK_HOME/platforms/android-26/android.jar" -S res -F bin/$1-unaligned.apk bin/libs
+    aapt package -f -M AndroidManifest.xml -I "$ANDROID_SDK_HOME/platforms/android-26/android.jar" -S res -F bin/$1-unaligned.apk
+    aapt add -f bin/$1-unaligned.apk bin/libs/*.so
```
Comment (juan-lunarg) [inline build-android/build_all.sh:77]: Add comments for lines 76 and 77 similar to the code below

**Review by juan-lunarg — CHANGES_REQUESTED** [review]
LGTM just needs comments

File: build-android/build_all.sh (hunk 1 above)
Comment (aitor-lunarg) [reply build-android/build_all.sh:77]: Yes, I'll wait for confirmation that this works and I'll update with comments

Comment (juan-lunarg) [reply build-android/build_all.sh:77]: Please also revise the commit message to be more specific as well.

**Comment by aitor-lunarg** [comment]
Won't solve issue

**Pull request closed without merging**